publications-2017.bib

```diff
@@ -1,5 +1,102 @@
+  Author                   = {Ganesan, S},
+  Year                     = {2017},
+  School = {University of Michigan},
+  Url                      = {http://www-personal.umich.edu/~veeras/papers/sriramthesis.pdf}
+}
+
+@PhdThesis{Giuliani2017,
+  Title                    = {{Modelling Fluid Structure Interaction problems using Boundary Element Method}},
+  Author                   = {Giuliani, N},
+  School                   = {Scuola Internazionale Superiore di Studi Avanzati - Trieste},
+  Year                     = {2017},
+  Url                      = {https://iris.sissa.it/handle/20.500.11767/57293}
+}
+
+@InProceedings{Goluchova,
+  Title                    = {Epicyclic oscillations of thick relativistic disks},
+  Author                   = {Goluchov{\'{a}}, K and T{\"{o}}r{\"{o}}k, G},
+  Booktitle                  = {Proceedings of RAGtime 17–19, 17–19/23–26 Oct., 1–5 Nov., 2015/2016/2017, Opava, Czech Republic},
+  Editor = {Z. Stuchl{\’i}k, G. Tor{\”o}k and V. Karas},
+  Pages = {47--59},
+  Url                      = {http://proceedings.physics.cz/images/proc17/hor2.pdf}
+}
+
+@article{Grohs2017,
+  doi = {10.5802/smai-jcm.26},
+  url = {https://doi.org/10.5802/smai-jcm.26},
+  year = {2017},
+  publisher = {Cellule {MathDoc}/{CEDRAM}},
+  volume = {3},
+  pages = {219--248},
+  author = {Philipp Grohs and Ralf Hiptmair and Simon Pintarelli},
+  title = {Tensor-Product Discretization for the Spatially Inhomogeneous and Transient Boltzmann Equation in Two Dimensions},
+  journal = {{SMAI} Journal of Computational Mathematics}
+}
+
+@PhDThesis{Holmgren2017,
+  Title                    = {{Modelling of Moving Contact Lines in Two-Phase Flows}},
+  Author                   = {Holmgren, H},
+  Year                     = {2017},
+  School = {Uppsala University},
+  Url                      = {http://www.diva-portal.org/smash/record.jsf?pid=diva2:1139929}
+}
+
+@InProceedings{Korous,
+  Title                    = {Distributed Implicit Discontinuous Galerkin {MHD} Solver},
+  Author                   = {Korous, L and Karban, P and Skala, J},
+  Booktitle = {Proceedings of the 2017 CompuMag conference},
+  Url                      = {http://www.compumag.org/CMAG2017/[PD-A7-13]_492.pdf}
+}
+
+@PhDThesis{Joos2017,
+  Title                    = {{Microstructural Characterisation, Modelling and Simulation of Solid Oxide Fuel Cell Cathodes}},
+  Author                   = {Joos, J},
+  Year                     = {2017},
+  School = {Karlsruhe Institute of Technology},
+  Url                      = {https://publikationen.bibliothek.kit.edu/1000064791/4189242}
+}
+
+@phdthesis{ubsi_1237,
+	author = {Andreas Kr{\”a}mer},
+	title = {Lattice-{B}oltzmann-{M}ethoden zur {S}imulation inkompressibler {W}irbelstr{\”o}mungen},
+	year = {2017},
+	school = {Universit{\”a}t Siegen},
+	booktitle = {Schriftenreihe des Lehrstuhls für Str{\”o}mungs- und Thermodynamik},	
```

Comment:
Should there be a book title with a phd thesis?

Reply:
Looks like so: https://d-nb.info/1144615712/34
Even if the book is published by the institute.

Reply:
No, `@phdthesis` doesn't allow this field: http://bib-it.sourceforge.net/help/fieldsAndEntryTypes.php#phdthesis